src/parser/core/Events.ts

```diff
@@ -96,6 +96,7 @@ export enum EventType {
   FeedHeal = 'feed_heal',
 
   // Warlock
+  Create = 'create',
```

Comment:
This is a generic non-warlock specific event wow has. It should be placed at the top and not under a specific class tag (very w/e since its just location in a enum)
Since I can't comment on un-edited lines you will also need to add this to MappedEventTypes which will require you to make a interface for it with the correct fields.